src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      class="form"
+      action="https://github.com/alexsinchuk/layout_search-bar-airbnb"
+      method="get"
+      data-qa="big"
+      >
```

Comment:
```suggestion
    <form
      class="form"
      action="https://github.com/alexsinchuk/layout_search-bar-airbnb"
      method="get"
      data-qa="big"
    >
```

Closing bracket has to be at the same level as opening